src/index.ts

```diff
@@ -81,6 +81,10 @@ export const handleTokenRequest = async (ctx: Context, request: Request) => {
 	const signedToken = await issuer.issue(tokenRequest);
 	ctx.metrics.signedTokenTotal.inc({ env: ctx.env.ENVIRONMENT });
```

Comment:
could you add `key_id: keyId` here instead? This is the same metric overall, just adding one label.

Reply:
Added the label + service label